packages/web-components/.storybook/preview-head.html

```diff
@@ -5,5 +5,6 @@
     margin: 8px;
     height: 100%;
     background: var(--fill-color);
+    color: var(--neutral-foreground-rest);
```

Comment:
@bheston I think we'll want to pull this as it hides places we should probably have a good default.

Reply:
The reason I added this is that the two go hand in hand. This is what the design-system-provider does, and the Storybook wrapper is essentially acting as the DSP by specifying the background color at all. I think this is safe because it's intended for plain text on the page, like the headings of the component samples. It should never apply into a component, but if we want a place to test something like that I think Storybook is not that place. If it is, we should also remove the background color here.

This question has come up a few times, and I think having a default foreground color is actually a good idea. Keeping it paired so that if a section were updated the foreground would change. That's how the design tokens would work now as well. It's the default way we can handle contrast requirements. Otherwise for dark mode testing this text is black on grey.

Reply:
I can understand this, but while it's not intended to apply to components, it often does, most often with instances of slotted content in the light DOM. This leads to places where we assume that something is styled because the native elements inherit - but I don't think we should assume that all app and website authors will always do this, or that it's desired.  

If the goal is to ensure that the common elements which make up the page outside the components theme accordingly, we can address that by swapping this for areas where we target most of those selectors - primarily headings. That can create issues if we're slotting heading content, but it's far less egregious than setting color on the body and having nearly every element inherit that color. That makes it very difficult to find edge cases where a good default could have been set or served.

If there were a way to ensure that non-light-DOM instances of text did not inherit this, then I would agree, but that's currently not the case. Given the way that CSS works and our goals for the storybook - a development site for testing the components and Adaptive UI, I think that headings theming are secondary and the implementation should play more towards our greatest needs for storybook. 

Given that, would it seem reasonable to update this to apply the color to *heading* elements rather than the body to avoid the inheritance issue while ensuring that the headings for stories show up as expected?

Reply:
I think the latest update does a reasonable job of catching the elements we expect without reaching into anything we don't.